datahub-web-react/src/conf/theme/types.ts

```diff
@@ -30,5 +30,10 @@ export type Theme = {
         search: {
             searchbarMessage: string;
         };
+        menu: {
+            label: string;
+            path: string;
+            shouldOpenInNewTab: boolean;
+        }[];
```

Comment:
can we make this type

```
        menu: {
            items: {
                label: string;
                path: string;
                shouldOpenInNewTab: boolean;
            }
        }[];
```

instead? the advantage is if we want to add menu properties aside from items later down the line, its easy to make that change

Reply:
I have made the changes for more  flexible customization as mentioned by you.